service/informer/informer_test.go

```diff
@@ -460,3 +469,7 @@ func (e *testEventer) FetchLatest(project, environment string) (eventerspec.Depl
 
 	return eventerspec.DeploymentEvent{}, eventer.NotFoundError
 }
+
+func (e *testEventer) SetPendingStatus(event eventerspec.DeploymentEvent) error {
+	return nil
```

Comment:
Due to time pressure I did not write tests for this now. Something for boyscouting later.